extensions/toml/src/toml.rs

```diff
@@ -58,7 +54,10 @@ impl TomlExtension {
         fs::create_dir_all(&version_dir)
             .map_err(|err| format!("failed to create directory '{version_dir}': {err}"))?;
 
+        #[cfg(not(windows))]
```

Comment:
We should use the extension API to detect the platform, not compile-time directives.

The Wasm bundles get built on Linux, so this will not accurately detect Windows.

Reply:
Oh, thanks for noted.